src/irmin-test/common.ml

```diff
@@ -34,7 +34,7 @@ let merge_exn msg x =
   | Error (`Conflict m) -> Alcotest.failf "%s: %s" msg m
 
 let info msg =
-  let date = Int64.of_float (Unix.gettimeofday ()) in
+  let date = Int64.of_float 0. in
```

Comment:
it's probably worth pushing this in a separate PR. The idea here was to have reproducible hashes on the tests.

Reply:
Done in https://github.com/mirage/irmin/pull/1390.